modules/auxiliary/admin/kerberos/ticket_converter.rb

```diff
@@ -0,0 +1,67 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Kerberos ticket converter',
+        'Description' => %q{
+          This module converts tickets to the ccache format from the kirbi format and vice versa.
+        },
+        'Author' => [
+          'Dean Welch', # Metasploit Module
+          'Zer1t0' # Impacket Implementation https://github.com/Zer1t0
+        ],
+        'References' => [
+          ['URL', 'https://github.com/SecureAuthCorp/impacket/blob/3c6713e309cae871d685fa443d3e21b7026a2155/examples/ticketConverter.py'],
+          ['URL', 'https://tools.ietf.org/html/rfc4120'],
+          ['URL', 'http://web.mit.edu/KERBEROS/krb5-devel/doc/formats/ccache_file_format.html'],
+          ['URL', 'https://github.com/gentilkiwi/kekeo'],
+          ['URL', 'https://github.com/rvazarkar/KrbCredExport'],
+        ],
+        'License' => MSF_LICENSE,
+        'Notes' => {
+          'Stability' => [],
+          'Reliability' => [],
+          'SideEffects' => []
+        }
+      )
+    )
+    register_options(
+      [
+        OptPath.new('InputFile', [ true, 'The file path to ticket in kirbi (KRB-CRED) or ccache format.' ]),
+        OptString.new('OutputPath', [ true, 'The output path to save the converted ticket.' ]),
```

Comment:
Little odd that the File and Path suffixes are mixed here. It'd be easier to remember if they were both the same.

```suggestion
        OptPath.new('InputPath', [ true, 'The file path to ticket in kirbi (KRB-CRED) or ccache format.' ]),
        OptString.new('OutputPath', [ true, 'The output path to save the converted ticket.' ]),
```